src/views/shared/transaction-info/transaction-info.view.jsx

```diff
@@ -91,13 +91,15 @@ function TransactionInfo({
               subtitle: "My Hermez address",
               value: getPartiallyHiddenHermezAddress(txData.fromHezEthereumAddress),
             }}
-            to={{
-              subtitle:
-                txData.toHezEthereumAddress?.toLowerCase() ===
-                INTERNAL_ACCOUNT_ETH_ADDR.toLowerCase()
-                  ? getPartiallyHiddenHermezAddress(txData.toBjj || txData.toBJJ)
-                  : getPartiallyHiddenHermezAddress(txData.toHezEthereumAddress),
-            }}
+            to={
+              txData.toHezEthereumAddress && {
```

Comment:
And what if it's a tx to an internal account and it has a BJJ but no ethereum address?

Reply:
The last code wasn't covering that scenario so I assume that it's rare to happen. This fix is covering a corner case where those values aren't coming. In theory should be return at least the value that was submitted.
At least with this fix doesn't crash the app in your case either

Reply:
But with this change, internal transactions will stop showing the To field, and we can't allow that.

Reply:
we can show the same address if it's the known case and cover for the unexpected